InventorAccessPortal.Web/Mailer/ResetPasswordController.cs

```diff
@@ -0,0 +1,211 @@
+﻿using System;
```

Comment:
Spilt this file into one that does the first view (reset password) then pass to the mailer